docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -71,38 +105,54 @@ CPAN].
 
 [ZNMSTR account at CPAN]: http://search.cpan.org/~znmstr/
 
-## 7. Verify that the module builds and all tests pass
+## 8. Verify that the module builds and all tests pass
+
+Find the list of supported Perl versions and locales in the [declaration of
+prerequisites]. Then find the latest point-release of each supported Perl
+version in the table of [latest releases in each branch of Perl].
+
+Make sure you've set up perlbrew and installed the latest point-release of each
+supported Perl version for perlbrew.
+
+    perlbrew init
+    perlbrew install perl-5.14.4  # etc
```

Comment:
perlbrew install perl-5.14.4  # For every version of Perl

Reply:
Making this update now. In the long term I'm not sure about the entire perlbrew thing since we have Travis. They kind of solve the same problem.